perun-core/src/test/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_member_attribute_def_def_memberOrganizationsTest.java

```diff
@@ -0,0 +1,68 @@
+package cz.metacentrum.perun.core.impl.modules.attributes;
+
+import cz.metacentrum.perun.core.api.Attribute;
+import cz.metacentrum.perun.core.api.Member;
+import cz.metacentrum.perun.core.api.Vo;
+import cz.metacentrum.perun.core.api.exceptions.VoNotExistsException;
+import cz.metacentrum.perun.core.api.exceptions.WrongReferenceAttributeValueException;
+import cz.metacentrum.perun.core.bl.PerunBl;
+import cz.metacentrum.perun.core.bl.VosManagerBl;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+
+
+public class urn_perun_member_attribute_def_def_memberOrganizationsTest {
+
+	private urn_perun_member_attribute_def_def_memberOrganizations classInstance;
+	private Attribute attributeToCheck;
+	private PerunSessionImpl session;
+	private Member member;
+	private Vo vo;
+
+	private ArrayList<String> memberOrganizations;
+
+	@Before
+	public void setUp() throws Exception {
+		classInstance = new urn_perun_member_attribute_def_def_memberOrganizations();
+		attributeToCheck = new Attribute(classInstance.getAttributeDefinition());
+
+		session = mock(PerunSessionImpl.class);
+		member = mock(Member.class);
+		vo = mock(Vo.class);
```

Comment:
I think it would be a little better if the vo object wasn't mocked but properly instantiated. Mocking would mean that its shortName is null (and I believe our list attributes never contain a null value).
```suggestion
		vo = new Vo(0, "Test VO", "test");
```